### DISCUSSION TITLE
Single station

### DISCUSSION THREAD

**Discussion by AlvicePC** [ROBelgium/MSNoise#272] [General]
I try to compute components for a single station but it doesn't create stack jobs when the command compute_cc finish, what I'm doing wrong, I inserted components to compute for a single station ZZ, ZE, ZN and set default components to compute but apparently it's not working good. Can somebody help me?

**Comment by ThomasLecocq** [comment]
could you try to remove the spaces between the elements of the list, and sort components alphabetically
``ZZ,EZ,NZ,EN``
?

**Reply by AlvicePC** [reply]
Hi Thomas, I tried it but it still not working, what are the parameters that I have to change to do a single station cross correlation? cc_type? may be there is my mistake because it don't create the 04 stack jobs folder for the 4th filter when finish the cc jobs.

**Reply by ThomasLecocq** [reply]
did you scan all the components of the data (Z, N, E components) = do they appear in the data_availability table/tab in the msnoise admin ?
in all cases ZZ autocorr should work (if you did run msnoise new_jobs after changing the parameter, of course, because without jobs, msnoise doesn't know it has to make jobs)...
